rest/src/main/java/com/carhouse/rest/response/ExceptionJSONResponse.java

```diff
@@ -0,0 +1,104 @@
+package com.carhouse.rest.response;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+
+/**
+ * The class is used to create response on the exceptions.
+ */
+@Component
+public class ExceptionJSONResponse {
```

Comment:
It can't be `ExceptionJSONResponse` cause response doesn't contain JSON at all

Reply:
ok